src/modules/predefinedDimensions.js

```diff
@@ -13,19 +13,19 @@ export const DIMENSION_PROP_NO_ITEMS = 'noItems'
 const FIXED_DIMENSIONS = {
     [DIMENSION_ID_DATA]: {
         id: DIMENSION_ID_DATA,
-        name: i18n.t('Data'),
+        getName: () => i18n.t('Data'),
```

Comment:
Note: Changes like these are actually breaking, since they're subsequently exported publicly through the likes of `getDimensionById` et al. Either mark this PR as a breaking change or make sure that the exposed API signature remains the same (by e.g. change the `getName()` prop back to `name` for the exposed functions.

Reply:
Yes, I'll mark it as a BREAKING CHANGE.

Reply:
After some tweaks, I was able to redo the implementation to avoid the breaking change (thanks for the tip!)